packages/next/cms/components/fields/ProfileSearch.js

```diff
@@ -277,6 +277,9 @@ export function ProfileSearch({
 
   const [debouncedQuery] = useDebouncedValue(query, 400);
   const profiles = useCMSProfiles();
+  const [inputRef, setInputFocus] = useFocus();
+
+  React.useLayoutEffect(() => setInputFocus(), [setInputFocus]);
```

Comment:
Shouldn't this go inside the `useFocus` hook? (And we could steal this for bespoke 🥷🏻)

Also, this will run on every re-render, as `setInputFocus` gets created on every function call. Do we want that?